index.html

```diff
@@ -967,8 +969,8 @@ <h2><dfn>Coalesced events</dfn></h2>
             that were coalesced into this event; otherwise it is an empty list.</p>
 
             <p>The events in the coalesced event list will have increasing
-            <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a>
-            ([[!WHATWG-DOM]]), so the first event will have the smallest <code>timeStamp</code>.</p>
+            {{Event/timeStamp}}s
+            ([[!WHATWG-DOM]]), so the first event will have the smallest {{Event/timeStamp}}.</p>
```

Comment:
General note to the Editors: It's best not to sprinkle [[SPEC]] everywhere. Sometimes things get moved to other specs, so it's best just to let ReSpec figure out where something is coming from (it will put the right thing in the bibliography). 


```suggestion
            {{Event/timeStamp}}s, so the first event will have the smallest {{Event/timeStamp}}.</p>
```